isis/src/base/objs/ShapeModel/ShapeModel.cpp

```diff
@@ -474,7 +504,7 @@ namespace Isis {
 
 
   /**
-   * Sets the normal for the currect intersection point.
+   * Sets the surface normal for the currect intersection point.
    * Note: This method will throw an error if this ShapeModel doesn't have
    * and intersection. Use the hasIntersection() method to verify before
```

Comment:
Spelling typo: "and intersection" to "an intersection"
Applies to the `setLocalNormal()` description below too